app/views/assignments/_list.html.erb

```diff
@@ -23,26 +23,41 @@
                   id: assignment.id %>
               </td>
               <td>
+                <% days = ((((assignment.due_date - Time.zone.now)/(60 * 60 * 24)).floor).to_s) %>
+                <% hours = ((((assignment.due_date - Time.zone.now)/(60 * 60)) % 24).floor) %>
```

Comment:
This mods out 24, so it won't work for displaying the total number of hours left when there's between 24-48 hours remaining.